go.mod

```diff
@@ -33,8 +33,8 @@ require (
 	k8s.io/apiextensions-apiserver v0.18.9
 	k8s.io/apimachinery v0.18.9
 	k8s.io/client-go v0.18.9
-	sigs.k8s.io/cluster-api v0.3.10
+	sigs.k8s.io/cluster-api v0.3.11
 	sigs.k8s.io/controller-runtime v0.6.4
 )
 
-replace sigs.k8s.io/cluster-api v0.3.10 => github.com/giantswarm/cluster-api v0.3.10-gs
+replace sigs.k8s.io/cluster-api v0.3.11 => github.com/giantswarm/cluster-api v0.3.10-gs
```

Comment:
Will take care of adding `v0.3.11-gs` I guess?
Not really sure about this.

Reply:
Might be better to stick with 0.3.10 since there can be relatively big changes in CAPI patch releases I think. We are going to stick with 0.3.10-gs until upstream supports Kubernetes 1.18+.